Projects/RevenueCat/Project.swift

```diff
@@ -0,0 +1,332 @@
+            destinations: allDestinations,
+            product: .unitTests,
+            bundleId: "com.revenuecat.BackendIntBackendCustomEntitlementsIntegrationTestsegrationTests",
+            deploymentTargets: .multiplatform(
+                iOS: "16.0",
+                macOS: "11.0",
+                watchOS: "7.0",
+                tvOS: "14.0"
+            ),
+            infoPlist: .default,
+            sources: [
+                "../../Tests/BackendIntegrationTests/CustomEntitlementsComputationIntegrationTests.swift",
+                "../../Tests/UnitTests/Misc/**/TestCase.swift",
+                "../../Tests/BackendIntegrationTests/BaseBackendIntegrationTests.swift",
+                "../../Tests/BackendIntegrationTests/BaseStoreKitIntegrationTests.swift"
+            ],
+            dependencies: [
+                .target(name: "RevenueCat"),
+                .target(name: "BackendIntegrationTestsHostApp"),
+                .nimble,
+                .ohHTTPStubsSwift,
+                .snapshotTesting,
+                .storeKitTests
+            ],
+        ),
+
+        .target(
+            name: "BackendIntegrationTests",
+            destinations: allDestinations,
+            product: .unitTests,
+            bundleId: "com.revenuecat.BackendIntegrationTests",
+            deploymentTargets: .multiplatform(
+                iOS: "16.0",
+                macOS: "11.0",
+                watchOS: "7.0",
+                tvOS: "14.0"
+            ),
+            infoPlist: .default,
+            sources: [
+                .glob(
+                    "../../Tests/BackendIntegrationTests/**/*.swift",
+                    excluding: [
+                        "../../Tests/BackendIntegrationTests/CustomEntitlementsComputationIntegrationTests.swift"
+                    ]
+                ),
+                "../../Tests/UnitTests/Mocks/MockSandboxEnvironmentDetector.swift",
+                "../../Tests/UnitTests/Misc/**/TestCase.swift",
+                "../../Tests/UnitTests/Misc/**/XCTestCase+Extensions.swift",
+                "../../Tests/UnitTests/TestHelpers/**/*.swift",
+                .glob(
+                    "../../Tests/StoreKitUnitTests/TestHelpers/**/*.swift",
+                    excluding: [
+                        "../../Tests/StoreKitUnitTests/TestHelpers/StoreKitConfigTestCase+Extensions.swift"
+                    ]
+                )
+            ],
+            resources: [
+                "../../Tests/StoreKitUnitTests/UnitTestsConfiguration.storekit"
+            ],
+            dependencies: [
+                .target(name: "RevenueCat"),
+                .target(name: "BackendIntegrationTestsHostApp"),
+                .nimble,
+                .ohHTTPStubsSwift,
+                .snapshotTesting,
+                .storeKitTests
+            ],
+            additionalFiles: [
+                "../../Tests/StoreKitUnitTests/UnitTestsConfiguration.storekit",
```

Comment:
we are using a different storekit file here `RevenueCat_IntegrationPurchaseTesterConfiguration.storekit`